utopia-remix/app/models/projectAccess.server.ts

```diff
@@ -29,28 +29,24 @@ export async function createProjectAccess(params: {
   projectId: string
   accessLevel: AccessLevel
 }): Promise<void> {
-  // check if access level already exists
-  const projectAccess = await prisma.projectAccess.findUnique({
-    where: {
-      project_id: params.projectId,
-    },
-  })
-  if (projectAccess != null) {
-    console.error(`Project access already exists for project ${params.projectId}`)
-    return
-  }
   await prisma.$transaction(async (tx) => {
-    await tx.projectAccess.upsert({
+    // check if project access already exists
+    const projectAccess = await prisma.projectAccess.findUnique({
```

Comment:
this needs to use `tx`

```suggestion
    const projectAccess = await tx.projectAccess.findUnique({
```